examples/gno.land/p/demo/grc/grc20/banker.gno

```diff
@@ -24,6 +25,8 @@ type Banker struct {
 	token       *token   // to share the same pointer
 }
 
+const uint64_max = 1<<64 - 1
```

Comment:
This is already defined as `math.MaxUint64` if you want to use that.

Reply:
commit 0356eca4